src_c/event.c

```diff
@@ -1901,6 +1919,14 @@ _pg_get_all_events(void)
         for (loop = 0; loop < len; loop++) {
             if (!_pg_event_append_to_list(list, &eventbuf[loop]))
                 goto error;
+
+            if (eventbuf[loop].type == SDL_KEYDOWN) {
+                pressed_keys[(int)eventbuf[loop].key.keysym.scancode] = 'T';
```

Comment:
Even though they are chars now you can and should use `0` and `1` still, no need for T and F